resources/l10n/de/LC_MESSAGES/loot.po

```diff
@@ -9,324 +9,363 @@ msgid ""
 msgstr ""
 "Project-Id-Version: LOOT 0.16.0\n"
```

Comment:
Please update the project version, see point 5 here:
https://github.com/loot/loot/blob/master/CONTRIBUTING.md#translating-the-loot-application

Reply:
Done